Uplift/Views/GymDetail/Facilities/MiscellaneousInfoView.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  MiscellaneousInfoView.swift
+//  Uplift
+//
+//  Created by Phillip OReggio on 10/27/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+class MiscellaneousInfoView: UIView {
+
+    private let textView = UITextView()
+
+    init(miscellaneousInfo: [String]) {
+        super.init(frame: CGRect.zero)
+
+        let display = formatMiscellaneous(misc: miscellaneousInfo)
+        textView.attributedText = generateAttributedString(for: display, font: UIFont._14MontserratRegular!)
+        textView.isEditable = false
+        textView.isSelectable = false
+        textView.isScrollEnabled = false
+        textView.textContainerInset = .zero
+        textView.textContainer.lineFragmentPadding = 0
+
+        addSubview(textView)
+        setupConstraints()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    func setupConstraints() {
+        textView.snp.makeConstraints { make in
+            make.edges.equalToSuperview()
+        }
+    }
+
+    // MARK: - Helper
+
+    func formatMiscellaneous(misc: [String]) -> String {
```

Comment:
Replace the `misc` with `_ misc` since there's only one argument and the function name already implies the argument's purpose.